src/js/entities-service/entities/program-actions.js

```diff
@@ -129,15 +129,25 @@ const Collection = BaseCollection.extend({
       data: JSON.stringify({ data }),
     });
   },
+  isVisibleToCurrentUser(action) {
+    const visibleToList = action.get('_visible_to');
+    const currentUser = Radio.request('bootstrap', 'currentUser');
+    const currentUserTeamId = currentUser.get('_team');
+
+    if (!size(visibleToList)) return true;
```

Comment:
If `relationships.visible_to` is `null`/`undefined` or an empty `[]`, assume it's visible to every team.